package.json

```diff
@@ -7,7 +7,7 @@
     "chai": "*",
     "eventemitter2": "~2.0.2",
     "google-auto-auth": "~0.2.4",
-    "grpc": "~0.15.0",
+    "grpc": "~1.0",
```

Comment:
Additionally, in the future, I would prefer that these sorts of changes be in separate commits to make rolling back easier if something breaks.